static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -0,0 +1,261 @@
+
+                    this.$els.header.css({
+                        position:  'relative',
+                        'margin-top': bannerHeight,
+                        'transform': 'translateY(-500%)',
+                        'z-index': '998'
+                    });
+
+                    this.$els.main.css('margin-top', this.headerBigHeight - this.$els.header.dim().height);
+                }
+            } else {
+                // Make sure that we show slim nav when page loaded with anchor
+                this.$els.bannerDesktop.css({
+                    position:  'fixed',
+                    top:       0,
+                    width:     '100%',
+                    'z-index': '999'
+                });
+                // Header is not slim yet
+                this.$els.header.removeClass('l-header--is-slim');
+
+                // Put navigation to its default state
+                this.setNavigationDefault();
+
+                this.$els.header.css({
+                    position:  'relative',
+                    width:     '100%',
+                    'margin-top': bannerHeight,
+                    'transform': 'translateY(0%)',
+                    'z-index': '998'
+                });
+
+                this.$els.main.css('margin-top', 0);
+            }
+
+        }.bind(this));
+    };
+
+    StickyHeader.prototype.updatePositionMobile = function () {
+        var bannerHeight = this.$els.bannerMobile.dim().height,
+            scrollY;
+
+        fastdom.read(function () {
+            scrollY = $(window).scrollTop();
+        });
+
+        fastdom.write(function () {
+            this.setScrollDirection(scrollY);
+
+            //header, navigation and banner are sticky from the beginning
+            if (scrollY < this.config.thresholdMobile) {
+                this.$els.header.css({
+                    position: 'fixed',
+                    top: 0,
+                    width: '100%',
+                    'z-index': '1001',
+                    'margin-top': 0
+                });
+                this.$els.bannerMobile.css({
+                    position:  'fixed',
+                    top:       this.headerBigHeight,
+                    width:     '100%',
+                    'z-index': '999' // Sticky z-index -1 as it should be sticky but should go below the sticky header
+                });
+                this.$els.main.css('margin-top', this.headerBigHeight + bannerHeight);
+
+                // Put navigation to its default state
+                this.setNavigationDefault();
+            } else {
+                //after this.thresholdMobile px of scrolling 'release' banner and navigation
+                this.$els.bannerMobile.css({
+                    position:  'absolute',
+                    top:       this.config.thresholdMobile + this.headerBigHeight
+                });
+                this.showNavigation(scrollY);
+            }
+        }.bind(this));
+    };
+
+    return new StickyHeader();
```

Comment:
I'm going to assume you just moved this file here rather than added it so won't bother reading it. Let me know if that's not the case